genai-perf/genai_perf/main.py

```diff
@@ -26,18 +26,22 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import os
+import signal
 import sys
+import threading
+import time
 import traceback
 
 import genai_perf.logging as logging
 from genai_perf import parser
 
+logger = logging.getLogger(__name__)
+
 
 # Separate function that can raise exceptions used for testing
 # to assert correct errors and messages.
 def run():
     # TMA-1900: refactor CLI handler
-    logging.init_logging()
```

Comment:
This should not have been deleted, it is needed for the logger to function properly

Reply:
Reverted, thanks for catching this.